dirty_cat/super_vectorizer.py

```diff
@@ -439,3 +439,14 @@ def get_feature_names_out(self, input_features=None) -> List[str]:
         get_feature_names.
         """
         return self.get_feature_names()
+
+    def get_transformers(self):
+        res  = []
+        for name, trans, cols in self.transformers_:
+            if name == "remainder":
+                res.append((name, trans,
+                            list(map(lambda i:self.columns_[i], cols))))
```

Comment:
List comprehensions are easier to read than pure functional programming for most people
```suggestion
                            [self.columns_[i] for i in cols])
```